ocaml/xapi/xapi_pool.ml

```diff
@@ -615,6 +615,44 @@ let protect_exn f x =
 		Debug.log_backtrace e (Backtrace.get e);
 		None
 
+let create_or_get_pvs_server_on_master __context rpc session_id pvs_site_ref (pvs_server_ref, pvs_server) : API.ref_PVS_server =
+	let my_uuid = pvs_server.API.pVS_server_uuid in
+	let new_pvs_server_ref =
```

Comment:
The name implies a new server gets created in every case but it can return an existing one. I'd use simply `server`. If you are using `uuid` you can also pass it as `~uuid` rather than `~uuid:my_uuid`.